.github/workflows/core.yml

```diff
@@ -74,10 +74,10 @@ jobs:
       - name: Checkout the Source Code
         uses: actions/checkout@v3
 
-      - name: Set Up Beta Clippy
+      - name: Set Up Clippy
         uses: ./.github/actions/setup-rust
         with:
-          toolchain: beta
+          toolchain: stable
```

Comment:
Not a big fan of this. I'd rather keep beta and temporarily allow if needed. Some clippy changes are pretty intense and may need some thinking about it. With stable we discover them too late.

Reply:
yeah, I wanted to try different options (which is why this is just a draft atm)